micro_brewery/demo/website_theme_apply.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+
+    <function model="ir.module.module" name="_theme_load" context="{'apply_new_theme': True}">
+        <value model="ir.module.module" eval="obj().env['ir.module.module'].search([('name', '=', 'theme_bistro')]).ids"/>
+        <value model="website" eval="obj().env.ref('website.default_website')"/>
+    </function>
+
+    <function model="ir.ui.view" name="write">
+        <value model="ir.ui.view" eval="obj().env['website'].with_context(website_id=obj().env.ref('website.default_website').id).viewref('website.homepage').id"/>
+        <value model="ir.ui.view" eval="{'arch': obj().env.ref('micro_brewery.homepage').arch}"/>
+    </function>
+
+    <function model="website.menu" name="write">
+        <value model="website.menu" eval="obj().search([('url', '=', '/shop'), ('website_id', '=', ref('website.default_website'))]).id"/>
+        <value eval="{'name': 'Our eShop'}"/>
+    </function>
+
+    <function model="website.menu" name="write">
+        <value model="website.menu" eval="obj().search([('url', '=', '/'), ('website_id', '=', ref('website.default_website'))]).id"/>
+        <value eval="{'name': 'About us', 'sequence': 1}"/>
+    </function>
+
+    <function model="website.menu" name="write">
+        <value model="website.menu" eval="obj().search([('url', '=', '/appointment'), ('website_id', '=', ref('website.default_website'))]).id"/>
+        <value eval="{'name': 'Visit our brewery', 'sequence': 4}"/>
+    </function>
+
+    <function model="ir.ui.view" name="write">
+        <value model="ir.ui.view" eval="obj().env['website'].with_context(website_id=obj().env.ref('website_sale.products_add_to_cart').id).viewref('website_sale.products_add_to_cart').id"/>
```

Comment:
Why you are adding template id., where it should be website_id in the context

Also you can direct use the ref, no need to use obj().env.ref